src/test/java/core/basesyntax/db/StorageTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.db;
+
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertSame;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class StorageTest {
+
+    @AfterEach
+    void tearDown() {
+        Storage.getFruitStorage().clear();
+    }
+
+    @Test
+    void getFruitStorage_getInstance_Ok() {
+        Map<String, Integer> storage = Storage.getFruitStorage();
+        assertNotNull(storage);
+        assertSame(storage, Storage.getFruitStorage());
+    }
+
+    @Test
+    void getFruitStorage_isHashMap_Ok() {
```

Comment:
In my opinion, tests like this one are unnecessary as they are too detailed - please focus on business details rather than technical ones.